Project64.Project64.Dev version 4.0.0.5713

techflashYT opened the pull request:
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----


###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/30994)

techflashYT committed:
Project64.Project64Dev version 4.0.0.5713

OfficialEsco reviewed (changes requested):
This Package will most likely cause issues with the stable version even with these additions.

techflashYT committed:
Update manifests/p/Project64/Project64Dev/4.0.0.5713/Project64.Project64Dev.installer.yaml

Co-authored-by: Esco <OfficialEsco@users.noreply.github.com>

techflashYT committed:
Update manifests/p/Project64/Project64Dev/4.0.0.5713/Project64.Project64Dev.installer.yaml

Co-authored-by: Esco <OfficialEsco@users.noreply.github.com>

techflashYT committed:
Update manifests/p/Project64/Project64Dev/4.0.0.5713/Project64.Project64Dev.locale.en-US.yaml

Co-authored-by: Esco <OfficialEsco@users.noreply.github.com>

techflashYT committed:
Update manifests/p/Project64/Project64Dev/4.0.0.5713/Project64.Project64Dev.locale.en-US.yaml

Co-authored-by: Esco <OfficialEsco@users.noreply.github.com>

techflashYT committed:
Update manifests/p/Project64/Project64Dev/4.0.0.5713/Project64.Project64Dev.yaml

Co-authored-by: Esco <OfficialEsco@users.noreply.github.com>

techflashYT committed:
Tried moving the folder

Moved from manifests/p/Project64/Project64Dev to manifests/p/Project64/Project64/Dev
